modules/button.scss

```
@@ -133,4 +123,48 @@
       }
     }
   }
+
+  @media (min-device-width: 320px) and (max-device-width: 428px) {
+    &--type--serch {
+      width: 158px;
+      height: 58px;
+    }
+
+    &--type--hotels {
+      font-size: 30px;
+      width: 100%;
+    }
+
+    &--type--icon {
+      width: 72px;
+      height: 72px;
+
+      & svg {
```

```suggestion
      & > svg {
```
Пока пусть будет атк, в будущем иконки станут тоже отдельный компонентом